backend/dev/user.clj

```diff
@@ -29,12 +30,28 @@
 (defn test []
   (eftest/run-tests (eftest/find-tests "test") {:multithread? false}))
 
-
 (defn migrate []
   (mig/migrate {:connection-uri (-> config :db :uri)}))
 
-(defn rollback []
-  (mig/rollback {:connection-uri (-> config :db :uri)}))
+(defn rollback [& args]
+  (mig/rollback {:connection-uri (-> config :db :uri)}
+                (map read-string args)))
+
+(defn- seed-tenant
+  "Helper function that will seed tenant to the tenants table."
+  [db tenant]
+  (jdbc/insert! db "tenants" tenant))
+
+(defn seed
```

Comment:
If you parameterize user/seed it could be used in fixtures/system-fixture replacing the insert-tenant call.